flink-connectors/flink-connector-kafka-0.10/src/test/java/org/apache/flink/streaming/connectors/kafka/KafkaTestEnvironmentImpl.java

```diff
@@ -229,6 +231,8 @@ public void prepare(int numKafkaServers, Properties additionalServerProperties,
 		standardProps.setProperty("zookeeper.connect", zookeeperConnectionString);
 		standardProps.setProperty("bootstrap.servers", brokerConnectionString);
 		standardProps.setProperty("group.id", "flink-tests");
+		standardProps.setProperty("key.deserializer", "org.apache.kafka.common.serialization.ByteArrayDeserializer");
+		standardProps.setProperty("value.deserializer", "org.apache.kafka.common.serialization.ByteArrayDeserializer");
```

Comment:
Somehow, without these settings, the new tests that test `setStartFromXXXX` methods will fail by complaining the property config does not specify settings for the deserializers.

I guess it is because in those tests, we have Kafka clients that are used *only* for offset committing and fetching, in which case the client cannot infer the types to use for the serializers?